back-end/src/routes/api.js

```diff
@@ -28,79 +33,86 @@ const uploadFile = (request, response) => {
 
       // Setting up S3 upload parameters
       const uploadParams = {
-          Bucket: BUCKET_NAME,
-          Key: newFileName, 
-          Body: fileContent
+        Bucket: BUCKET_NAME,
+        Key: newFileName, 
+        Body: fileContent
       };
       try {
         // Upload File to S3
         const uploadData = await s3.upload(uploadParams).promise();
         statusCode = 500;
         message = "Could not upload";
         if (uploadData) {
+          // eslint-disable-next-line
           console.log("File successfully uploaded")
           // Setting up rekognition parameters
           var rekogParams = {
             Image: {
-            Bytes: fileContent
+              Bytes: fileContent
             }, 
             MaxLabels: 10, 
             MinConfidence: 70
           };
 
           // Detect labels with Rekognition
           const rekogData = await rekognition.detectLabels(rekogParams).promise();
-          console.log("Labels detected")
-          console.log(rekogData)
+          // eslint-disable-next-line
+          console.log("Labels detected");
+          // eslint-disable-next-line
+          console.log(rekogData);
           const labels = rekogData.Labels;
-          var labelNames = []
+          var labelNames = [];
           labels.forEach( async (label) => {
             // Put names of labels inside labelNames
-            labelNames.push(label.Name)
+            labelNames.push(label.Name);
     
             // Set up dynamoDB params
             var tagParams = {
               TableName:"Tags",
               Item:{
-                  "tag": label.Name,
-                  "file": newFileName
+                "tag": label.Name,
+                "file": newFileName
               }
             };
     
             // Put tag item in table so we can query for files based on tags.
             const tagData = await dynamodb.put(tagParams).promise();
+            // eslint-disable-next-line
             console.log("Added Item: ", tagData);
           });
 
           // Set up parameters for putting document in files table.
           var fileParams = {
             TableName:"Files",
             Item: {
-                "file": newFileName,
-                "tags": labelNames,
-                "desc": request.body.desc,
-                "email": request.body.email //Change to request.user.email when authentication is set up
+              "file": newFileName,
+              "tags": labelNames,
+              "desc": request.body.desc,
+              "email": request.body.email //Change to request.user.email when authentication is set up
             }
           };
 
           const fileData = await dynamodb.put(fileParams).promise();
+          // eslint-disable-next-line
```

Comment:
please get rid of these disable comments. instead, add to the [.eslintrc.json file](https://github.com/evanugarte/SmartPhotoCategorizer/blob/master/.eslintrc.json).

change all occurances of `console.log` to `console.debug` and change [this line](https://github.com/evanugarte/SmartPhotoCategorizer/blob/06ab716b9a476a201c732b616a499d73fd0d6f08/.eslintrc.json#L49) to
`"no-console": ["error", { "allow": ["debug", "error"] }]`

Reply:
done